src/hyperfine/benchmark.rs

```diff
@@ -136,6 +137,7 @@ pub fn run_benchmark(
     cmd: &str,
     shell_spawning_time: TimingResult,
     options: &HyperfineOptions,
+    exporter: &mut Option<Box<ExportManager>>,
```

Comment:
It would be great if we could change the API a little bit. Instead of `run_benchmark` taking a mutable `ExportManager` and calling `add_result` on it, couldn't the function simply return the results (The `ExportEntry`)?